### PR TITLE
[FIX-IMP] stock_quantity_history_location: backport from v12 improvements/fixes

### PR DESCRIPTION
**Pull request by chienandalu**
Backport of improvements and fixes made on v11

cc @Tecnativa TT25978

### CODE REVIEW

File: stock_quantity_history_location/README.rst [hunk 1]
```diff
@@ -14,13 +14,13 @@ Stock Quantity History Location
...
-    :target: https://github.com/OCA/stock-logistics-reporting/tree/11.0/stock_quantity_history_location
+    :target: https://github.com/OCA/stock-logistics-reporting/tree/12.0/stock_quantity_history_location
     :alt: OCA/stock-logistics-reporting
 .. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
-    :target: https://translation.odoo-community.org/projects/stock-logistics-reporting-11-0/stock-logistics-reporting-11-0-stock_quantity_history_location
+    :target: https://translation.odoo-community.org/projects/stock-logistics-reporting-12-0/stock-logistics-reporting-12-0-stock_quantity_history_location
     :alt: Translate me on Weblate
 .. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
-    :target: https://runbot.odoo-community.org/runbot/151/11.0
+    :target: https://runbot.odoo-community.org/runbot/151/12.0
```
Comment (pedrobaeza) [inline stock_quantity_history_location/README.rst]: Don't change this

Comment (chienandalu) [reply stock_quantity_history_location/README.rst]: Should be ok now

**Review by pedrobaeza — APPROVED** [review]

**Comment by chienandalu** [comment]
@HviorForgeFlow are you ok with this?

**Review by victoralmau — APPROVED** [review]

**Pull request merged**